### ISSUE TITLE
Fix two's compliment

### ISSUE THREAD

**Issue by LloydLabs** [LloydLabs/librini#3]
We need to fix the two's compliment which occurs within the `rini_signed_int_str`, for some reason, when unit tests were being taken this was returning `0` on some cases.

**Comment by rudreais** [comment]
sorry, what ? `compliment` ?

**Comment by LloydLabs** [comment]
@rudreais `complement`, sorry about the typo.

**Comment by rudreais** [comment]
but you know that `rini_signed_int_str` isn't in any of your file ?

**Comment by LloydLabs** [comment]
Yes, this issue should be closed however I'm keeping it open as I might reimplement this. This is not even used within librini any more. 

@rudreais - hence please refrain from keep commenting on it, thanks!
